orc.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: orc
+  version: 0.4.34
+  epoch: 0
+  description: Oil Run-time Compiler
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - meson
+      - linux-headers
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha512: ea6ce72398fad7569b97afe8e635402a7cc6d5e27b57c27fd76d85ccdf971391b52187297e5849e50ce1031691e9748f2cc312cb39bd0c6ac23c00ac6d96cb41
+      uri: https://gstreamer.freedesktop.org/src/orc/orc-${{package.version}}.tar.xz
+
+  - uses: meson/configure
+    with:
+      opts: |
+        -Db_lto=true \
+        -Dorc-test=disabled
+
+  - uses: meson/compile
+
+  - uses: meson/install
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-compiler
+    description: ${{package.name}} compiler
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/bin
+          mv ${{targets.destdir}}/usr/bin ${{targets.subpkgdir}}/usr/bin
+
+  - name: ${{package.name}}-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
```

Comment:
The -dev packages should discover their dependencies using SCA since Melange 0.3 release.  We should not need these blocks anymore.